src/mainboard/clevo/tgl-u/variants/ns50mu/devicetree.cb

```diff
@@ -263,75 +255,65 @@ chip soc/intel/tigerlake
+			register "PcieRpLtrEnable[5]" = "1"
+			register "PcieClkSrcUsage[2]" = "5"
 			register "PcieClkSrcClkReq[2]" = "2"
-			chip drivers/gfx/nvidia/optimus
-				register "desc" = ""DGPU""
-				register "enable_gpio" = "ACPI_GPIO_OUTPUT_ACTIVE_HIGH(GPP_U5)" # DGPU_PWR_EN
-				register "reset_gpio" = "ACPI_GPIO_OUTPUT_ACTIVE_LOW(GPP_U4)" # DGPU_RST#_PCH
-				register "enable_delay_ms" = "16"
-				register "enable_off_delay_ms" = "4"
-				register "reset_delay_ms" = "10"
-				register "reset_off_delay_ms" = "4"
-				register "srcclk_pin" = "2" # PEG_CLKREQ#
-				register "ec_notify_method" = ""\\_SB.PCI0.LPCB.EC0.DGPM""
-				device generic 0 on end
-			end
 		end
-		device ref pcie_rp9 on
-			# PCIe root port #9 x1, Clock 3 (CARD)
-			register "PcieRpEnable[8]" = "1"
-			register "PcieRpLtrEnable[8]" = "1"
-			register "PcieClkSrcUsage[3]" = "8"
+		device ref pcie_rp7 on
+			# PCIe root port #7 x1, Clock 3 (GLAN)
+			register "PcieRpEnable[6]" = "1"
+			register "PcieRpLtrEnable[6]" = "1"
+			register "PcieClkSrcUsage[3]" = "6"
 			register "PcieClkSrcClkReq[3]" = "3"
-		end
-		device ref pcie_rp10 on
-			# PCIe root port #10 x1, Clock 4 (GLAN)
-			register "PcieRpEnable[9]" = "1"
-			register "PcieRpLtrEnable[9]" = "1"
-			register "PcieClkSrcUsage[4]" = "9"
-			register "PcieClkSrcClkReq[4]" = "4"
 			chip soc/intel/common/block/pcie/rtd3
 				register "enable_gpio" = "ACPI_GPIO_OUTPUT_ACTIVE_HIGH(GPP_F9)" # GPIO_LAN_EN
 				register "reset_gpio" = "ACPI_GPIO_OUTPUT_ACTIVE_LOW(GPP_C22)" # LAN_PLT_RST
-				register "srcclk_pin" = "4" # LAN_CLKREQ#
+				register "srcclk_pin" = "3" # GLAN_CLKREQ#
 				device generic 0 on end
 			end
 		end
-		device ref pcie_rp11 on
-			# PCIe root port #11 x1, Clock 1 (WLAN)
-			register "PcieRpEnable[10]" = "1"
-			register "PcieRpLtrEnable[10]" = "1"
-			register "PcieClkSrcUsage[1]" = "10"
+		device ref pcie_rp8 on
+			# PCIe root port #8 x1, Clock 1 (WLAN)
+			register "PcieRpEnable[7]" = "1"
+			register "PcieRpLtrEnable[7]" = "1"
+			register "PcieClkSrcUsage[1]" = "7"
 			register "PcieClkSrcClkReq[1]" = "1"
 			chip drivers/wifi/generic
 				register "wake" = "GPE0_DW0_23" # GPP_C23
 				device pci 00.0 on end
 			end
 		end
+		device ref pcie_rp9 on
+			# PCIe root port #9 x4, Clock 4 (SSD0)
+			register "PcieRpEnable[8]" = "1"
+			register "PcieRpLtrEnable[8]" = "1"
+			register "PcieClkSrcUsage[4]" = "8"
+			register "PcieClkSrcClkReq[4]" = "4"
+
+			chip soc/intel/common/block/pcie/rtd3
+				register "desc" = ""SSD0""
+				register "enable_gpio" = "ACPI_GPIO_OUTPUT_ACTIVE_HIGH(GPP_D14)"
+				register "reset_gpio" = "ACPI_GPIO_OUTPUT_ACTIVE_LOW(GPP_H0)"
+				register "srcclk_pin" = "4" # SSD0_CLKREQ#
+				register "is_storage" = "1"
+				register "cpu_pcie_clk_usage" = "8"
+				device generic 0 on end
+			end
+		end
 		device ref pch_espi on
 			chip ec/clevo/it5570
 				device pnp 0c09.0 on end
-				register "fan_mode" = "FAN_MODE_CUSTOM"
-				register "fans[0].curve.temperature" = "{ 40, 60, 70, 80 }"
```

Comment:
Why do we remove the fan curve?

Reply:
We don't know what values to use yet, this will probably need some testing

Reply:
I would keep it as is to avoid any complains the fans are too loud or do not comply with requested earlier policy.